src/app/poll/poll.component.html

```diff
@@ -430,18 +430,19 @@ <h1 class="text-center" data-id="pollHeading">
         } @else {
           <div class="row g-0 justify-content-end align-items-center">
             @if (formHelper.hasParticipantVotings()) {
-              <div
-                class="col-3 d-inline-flex align-items-center justify-content-end text-right mr-4">
+              <div class="col-3 d-inline-flex align-items-center justify-content-end text-right mr-4">
                 <img aria-hidden="true" alt="" class="circle-tag" src="assets/status_accepted.svg">
-                <div class="font-weight-bold voting-status-summary-added-participants pe-3" data-id="addedParticipation">
-                  {{ formHelper.getNumberOfVotingsWithVotingStatusAcceptedOfAddedParticipant() | leadingZero: 2 }}
+                <div id="added-participation" class="font-weight-bold pe-3" data-id="addedParticipation">
+                  <label class="visually-hidden" id="added-participation-label">{{ 'poll.acceptedTotal' | translate }}</label>
+                  {{ formHelper.getNumberOfVotingsWithVotingStatusAcceptedOfParticipant() | leadingZero: 2 }}
```

Comment:
```suggestion
                  {{ formHelper.getNumberOfVotingsWithVotingStatusAcceptedOfParticipant() }}
```

Ich weiß ehrlich gesagt nicht mehr, wieso wir hier führende Nullen haben...